app/views/macros/search-filters.njk

```diff
@@ -26,6 +26,9 @@
 					{% if object.selected %}
 						<div id='{{ key | lower }}Selected' class='govuk-!-font-size-14 govuk-!-margin-1 govuk-!-margin-left-4 nojs-hidden'>{{object.selected}} Selected</div>
 					{% endif %}
+					{% if object.name === 'Year' %}
+					  <span>Financial Year(1 April to 31 March)</span>
```

Comment:
The text is a little off compared to designs:
- Year should be year without the capital
- Please leave a space between year and (
- Reduce the overall size of the text to either 17 or 18px so it fits in one line

Reply:
Thanks FYC, Done!